src/test/fuzz/fuzz.cpp

```diff
@@ -151,8 +151,8 @@ void signal_handler(int signal)
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size)
 {
     static const auto& test_one_input = *Assert(g_test_one_input);
-    test_one_input({data, size});
-    return 0;
+    /* Returning -1 means the input was not useful. */
+    return int{test_one_input({data, size})} - 1;
```

Comment:
Just noting that this was only recently added to libFuzzer: https://reviews.llvm.org/D128749?id=441094

I think that is fine, running with older versions of libFuzzer makes little sense anyway.